2021-08-01T11:47:08.8830665+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T11:47:08.8909485+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T11:47:08.8910579+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T12:09:06.9262645+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T13:32:41.7438741+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T13:32:41.7500758+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T13:32:41.7501687+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T15:09:40.8849179+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T15:20:09.6437655+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T15:20:09.6508349+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T15:20:09.6509567+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T15:41:08.7861313+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T16:05:19.2853442+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T16:05:19.2913794+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T16:05:19.2915557+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T16:51:18.4453679+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T17:12:57.3834395+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T17:12:57.3897151+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T17:12:57.3898115+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T17:13:16.7957853+07:00 8000045b-0001-fd00-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T17:15:28.9145380+07:00 80000b00-0002-fb00-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T17:15:56.8544860+07:00 8000072a-0000-f700-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T17:16:13.3043646+07:00 800000f7-0001-f300-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T17:36:56.5181896+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T19:03:29.6137117+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T19:03:29.6198588+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T19:03:29.6199433+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T19:03:47.9742339+07:00 80000ae4-0002-fa00-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T19:04:26.7957810+07:00 800008c4-0000-f500-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T19:04:33.2118946+07:00 800000fe-0001-f300-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T19:04:46.4549768+07:00 80000482-0001-fd00-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T19:36:28.7693340+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T19:44:39.4957541+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T19:44:39.5028132+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T19:44:39.5029524+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T19:45:18.1543336+07:00 80000489-0001-fd00-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T19:48:24.7069671+07:00 8000098c-0001-fb00-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T19:51:19.8015404+07:00 80000297-0000-fa00-b63f-84710c7967bb [INF] PrintReport HomeControllerCould not find file 'G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\Reports\rptKetQua.repx'. (2d4508ee)
2021-08-01T20:11:20.3949098+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T20:12:21.3432972+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T20:12:21.3503842+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T20:12:21.3505385+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T20:12:31.7175119+07:00 800002fe-0000-f800-b63f-84710c7967bb [INF] G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\ReportPrint\20210801_081230_948_PM.pdf (574179fa)
2021-08-01T20:12:54.3417543+07:00 80000b0d-0002-fa00-b63f-84710c7967bb [INF] G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108\wwwroot\ReportPrint\20210801_081254_317_PM.pdf (c46a05e0)
2021-08-01T20:33:20.4922723+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T21:03:04.8166874+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T21:03:04.8240059+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T21:03:04.8241574+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T21:03:05.2159266+07:00 80000306-0000-f800-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Home__GetListKetQuaChuaIn.ExecuteAsync() in G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\InKetQua\Views\Home\_GetListKetQuaChuaIn.cshtml:line 90
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.<Invoke>g__AwaitMatcher|8_0(EndpointRoutingMiddleware middleware, HttpContext httpContext, Task`1 matcherTask)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-08-01T21:03:05.2299833+07:00 80000b1e-0002-fa00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Home__GetListKetQuaChuaIn.ExecuteAsync() in G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\InKetQua\Views\Home\_GetListKetQuaChuaIn.cshtml:line 90
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-08-01T21:03:09.1827854+07:00 80000b1f-0002-fa00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Home__GetListKetQuaChuaIn.ExecuteAsync() in G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\InKetQua\Views\Home\_GetListKetQuaChuaIn.cshtml:line 90
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-08-01T21:04:25.5774184+07:00 800005f8-0002-fd00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Home__GetListKetQuaChuaIn.ExecuteAsync() in G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\InKetQua\Views\Home\_GetListKetQuaChuaIn.cshtml:line 90
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-08-01T21:25:03.9669328+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T22:17:02.6880908+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T22:17:02.6942563+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T22:17:02.6943467+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T22:17:03.2314434+07:00 80000c6b-0003-f300-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Home__GetListKetQuaChuaIn.ExecuteAsync() in G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\InKetQua\Views\Home\_GetListKetQuaChuaIn.cshtml:line 90
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-08-01T22:32:16.0090408+07:00 80000112-0001-f300-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Home__GetListKetQuaChuaIn.ExecuteAsync() in G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\InKetQua\Views\Home\_GetListKetQuaChuaIn.cshtml:line 90
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-08-01T22:33:34.0317771+07:00 80000114-0001-f300-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Home__GetListKetQuaChuaIn.ExecuteAsync() in G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\InKetQua\Views\Home\_GetListKetQuaChuaIn.cshtml:line 90
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-08-01T22:34:54.6412378+07:00  [INF] Application is shutting down... (90f52c38)
2021-08-01T22:35:38.9635490+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-08-01T22:35:38.9726717+07:00  [INF] Hosting environment: "Production" (c3307c92)
2021-08-01T22:35:38.9728043+07:00  [INF] Content root path: "G:\SourceCodeWebApp\XNTestNhanhCovid\InKetQua\published0108" (b5d60022)
2021-08-01T23:07:54.7260651+07:00  [INF] Application is shutting down... (90f52c38)
